_src/pattern-library/sass/utilities/_variables.scss

```diff
@@ -331,6 +332,46 @@ $buttons-elevated-shadow: 3px !default;
 $buttons-border-radius: 3px !default;
 $buttons-font-weight: font-weight(semi-bold) !default;
 
+// forms
+$label-color: $text-base-color;
+$label-color-active: palette(grayscale, x-dark);
+
+$field-default-background: palette(grayscale, white);
```

Comment:
I'd prefer that we have these variable names follow the convention of [element]-[property]-[state]+[state]. So, ``field-default-background-focus`` becomes ``field-background-default+focus``.